ush/config.sh.RTMA_CONUS_3km

```diff
@@ -178,6 +178,7 @@ NCORES_PER_NODE=40 #kjet
 FG_ROOTDIR="/lfs4/BMC/nrtrr/NCO_dirs/nwges/para/RRFS_CONUS" #only needed by RTMA
 DT_ATMOS=1
 #adjust to low priority queue and possible preemption, slurm socket error, etc
+MAXTRIES_RUN_PREPSTART="3"
```

Comment:
Do we run this prepstart task?

Reply:
Yes, we run this task at EMC. If not used at GSL, we could move this to the WCOSS block.

Reply:
It is fine. Just to check it is used. Thanks!